scripts/webpack/plugins/FeatureFlaggedSriPlugin.js

```diff
@@ -0,0 +1,66 @@
+// @ts-check
+const webpack = require('webpack');
+
+/** @typedef {import('webpack/lib/Compiler.js')} Compiler */
+
+const PLUGIN_NAME = 'FeatureFlaggedSRIPlugin';
+const FEATURE_TOGGLE_WRAP = [
+  'if (window.grafanaBootData && window.grafanaBootData.settings && window.grafanaBootData.settings.featureToggles && window.grafanaBootData.settings.featureToggles.assetSriChecks) {',
+  '}',
+];
+
+/**
+ * Webpack plugin that wraps Webpack runtime integrity checks in a feature flag
+ * This allows us to disable SRI checks in both the initial chunks but also in the
+ * dynamically loaded chunks.
+ */
+class FeatureFlaggedSRIPlugin {
```

Comment:
This webpack plugin should be considered a temporary solution to only enabling subresource integrity in the webpack dynamically loaded chunks if the feature flag is enabled. After we've rolled this out and tested it across HG and on prem we can delete this plugin along with the feature flag when the time comes.